app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -1074,9 +1084,25 @@ class MainActivity : AppCompatActivity(), MainViewController,
         routeBtmContainer.visibility = View.VISIBLE
         distanceView.distanceInMeters = routeManager.route?.getTotalDistance() as Int
         destinationNameTextView.text = simpleFeature.name()
-        previewDirectionListView.adapter = DirectionListAdapter(this, instructionStrings,
-                instructionTypes, instructionDistances, routeManager.reverse,
-                MultiModalHelper(routeManager.route?.rawRoute))
+        if (routeManager.type == Router.Type.MULTIMODAL) {
```

Comment:
Next time we update the route preview logic we should look into moving the code to show/hide the instruction list into `RoutePreviewView` if possible.